src/routes/user.js

```diff
@@ -6,7 +6,7 @@ const { hashPassword, comparePassword } = require("../utils/helpers");
 //Handle logins:
 //Incoming: email/password 
 //Outgoing: response code or username/firstName/lastName
-router.post("/login", async (request, response) => {
+router.get("/login", async (request, response) => {
```

Comment:
Is there a reason why we are switching to get for login?

Reply:
I think this one was something I was working with that I forgot to take back out. It can just be put back to POST

Reply:
Got it. If you send out another commit it should update.